charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,104 @@
+# Ingest Support
+
+## Description
+
+The Ingest Support team is the liaison between the Data Coordination Platform (DCP) and labs contributing data to the Human Cell Atlas (HCA). This team closely collaborates with each data contributor to:
+
+*	Working with data contributors who are generating new data types to develop a deep understanding of their data
+* Assisting data contributors to describe their experiments in a manner that supports finability, reusability and reproducability with in the DCP ecosystem
+* Helping data contributors to submit their data to the DCP, resolving any emerging issues during the process
+
+## Definitions
+
+*Data contributor* - any user who uploads data and metadata into the DCP
+
+*Data wrangler* - a member of the ingest support team who assists to data contributors wanting to upload data into the DCP
+
+*Submission* - the grouping of data and metadata that makes sense to a data contributor. For example, a data contributor might typically upload all the data at once for a publication they have in preparation. This grouping does not necessarily correspond to how data is stored or displayed in the DCP and how it should be displayed to a data consumer
+
+*Data* - raw data files from an experiment such as fastq files for sequencing experiments or tiff stacks for imaging experiments
+
+*Metadata* - descriptive information about what projects, samples and experiments are and how they were collected or conducted
+
+*New data types* - experimental data types which either can't already be ingested by the DCP or have only recently (~6m - 1yr) started to be ingested by the system
+
+*Established data types* - experimental data types which have a history of submission and downstream use in the DCP and there are already many datasets of this type represented in the platform
+
+## Objectives
+
+- Assisting data contributors uploading data and metadata to the DCP
+- Helping data contributors fix errors with their data and metadata before it can be submitted to the DCP
```

Comment:
I think this was mentioned somewhere else in the document, but it might sound better to say "Helping data contributors _address_ errors with their data and metadata" instead of "Helping data contributors _fix_ errors with their data and metadata"

Reply:
I think I made this edit consistently based on Brian's recommendation but please do check